create/create.go

```diff
@@ -0,0 +1,139 @@
+package create
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	git "github.com/libgit2/git2go/v30"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/project"
+	"github.com/tarantool/cartridge-cli/templates"
+)
+
+const (
+	initialTagName   = "0.1.0"
+	initialCommitMsg = "Initial commit"
+)
+
+func CreateProject(projectCtx project.ProjectCtx) error {
+	log.Infof("Creating an application %q...", projectCtx.Name)
+
+	// check that application doesn't exist
+	if _, err := os.Stat(projectCtx.Path); err == nil {
+		return fmt.Errorf("Application already exists in %s", projectCtx.Path)
+	}
+
+	var err error
+
+	err = os.Mkdir(projectCtx.Path, 0755)
+	if err != nil {
+		return fmt.Errorf("Failed to create application directory: %s", err)
+	}
+
+	err = templates.Instantiate(projectCtx)
+	if err != nil {
+		os.RemoveAll(projectCtx.Path)
+		return fmt.Errorf("Failed to instantiate application template: %s", err)
+	}
+
+	log.Infof("Instantiated application files")
+
+	err = initGitRepo(projectCtx)
+	if err != nil {
+		log.Warnf("Failed to initialize git repo: %s", err)
+	} else {
+		log.Infof("Initialized git repo")
+	}
+
+	log.Infof("Application %q created successfully", projectCtx.Name)
+
+	return nil
+}
+
+func initGitRepo(projectCtx project.ProjectCtx) error {
+	var err error
+
+	// check that git is installed
+	if _, err = exec.LookPath("git"); err != nil {
```

Comment:
Does "git" that you use library supports such checks?

Reply:
I didn't find it in a documentation, but maybe it's possible.